app/adapters/organisations_adapter.rb

```diff
@@ -0,0 +1,11 @@
+class OrganisationsAdapter
+  def find(slug)
+    response = OrganisationFetcher.fetch(slug)
+    Organisation.new(
+      title: response["title"],
```

Comment:
Minor: I see these lines have been copied from `app/exporters/manual_publishing_api_exporter.rb` but as there's no interpolation happening I think I prefer single quotes (`'`) to double (`"`).